### ISSUE TITLE
With `ui.sidebar_menu_foldable = true` enabled keyboard focus moves to hidden menu items

### ISSUE THREAD

**Issue by jasonday** [google/docsy#1022]
Current behavior:
With `ui.sidebar_menu_foldable = true` enabled, when using the keyboard only, focus is placed on submenu items that are not visible. 

Expected behavior:
Only visible elements in the menu are able to receive focus.

**Comment by raum51** [comment]
I'll take a look on this issue (hopefully next week).

**Comment by kimberleybrown** [comment]
Bumping this topic up to see if a fix was applied for this. We recently had an accessibility audit and this issue in particular was still being flagged.

**Comment by jasonday** [comment]
yes this was merged - https://github.com/google/docsy/pull/1028

**Comment by chalin** [comment]
Is this closed by #1028?

**Comment by jasonday** [comment]
Yes. Verified code at line 151 - https://github.com/google/docsy/blob/main/assets/scss/_nav.scss#L151
